crates/vm/src/system/public_values/core.rs

```diff
@@ -42,6 +42,10 @@ impl<F: Field> BaseAir<F> for PublicValuesCoreAir {
     fn width(&self) -> usize {
         3 + self.encoder.width()
     }
+
+    fn columns(&self) -> Option<Vec<String>> {
+        todo!("Vec<T> does not implement StructReflection (PublicValuesCoreCols)")
```

Comment:
I think returning `None` would be better, so we can still export the PIL without column names.